docs/data/data-grid/list-view/ListView.tsx

```diff
@@ -0,0 +1,348 @@
+              />
+              {params.value}
+            </Stack>
+          );
+        },
+      },
+      {
+        field: 'createdAt',
+        headerName: 'Added',
+        type: 'date',
+        width: 200,
+        valueFormatter: formatDate,
+      },
+      {
+        field: 'updatedAt',
+        headerName: 'Modified',
+        type: 'date',
+        width: 200,
+        valueFormatter: formatDate,
+      },
+      {
+        field: 'type',
+        headerName: 'Type',
+        width: 150,
+      },
+      {
+        field: 'size',
+        headerName: 'Size',
+        width: 120,
+        valueFormatter: formatSize,
+      },
+      {
+        type: 'actions',
+        field: 'actions',
+        resizable: false,
+        width: 50,
+        getActions: (params) =>
+          isListView
+            ? [
+                <ActionDrawer
+                  params={params}
+                  container={container}
+                  onSaveRename={(value) => handleUpdate(params.id, 'name', value)}
+                  onDelete={() => handleDelete([params.id])}
+                  onOpen={() => {
+                    setDetailsState({ open: true, params });
+                  }}
+                />,
+              ]
+            : [
+                <GridActionsCellItem
+                  label="Open"
+                  icon={<OpenIcon fontSize="small" />}
+                  onClick={() => {
+                    setDetailsState({ open: true, params });
+                  }}
+                  showInMenu
+                />,
+                <GridActionsCellItem
+                  label="Rename"
+                  icon={<EditIcon fontSize="small" />}
+                  onClick={() =>
+                    apiRef.current?.startCellEditMode({
+                      id: params.id,
+                      field: 'name',
+                    })
+                  }
+                  showInMenu
+                />,
+                <GridActionsCellItem
+                  label="Delete"
+                  icon={<DeleteIcon fontSize="small" />}
+                  onClick={() => handleDelete([params.id])}
+                  showInMenu
+                />,
+              ],
+      },
+    ],
+    [isListView, handleDelete, handleUpdate],
+  );
+
+  const getEstimatedRowHeight = () => {
+    const density = apiRef.current.state.density;
+
+    if (isListView) {
+      switch (density) {
+        case 'compact':
+          return 47;
+        case 'standard':
+          return 67;
+        case 'comfortable':
+          return 97;
+        default:
+          return 67;
+      }
+    } else {
+      switch (density) {
+        case 'compact':
+          return 47;
+        case 'standard':
+          return 55;
+        case 'comfortable':
+          return 63;
+        default:
+          return 55;
+      }
+    }
+  };
+
+  return (
+    <div style={{ position: 'fixed', inset: 0 }}>
```

Comment:
```suggestion
    <div>
      <CSSBaseline />
```

Should be unnecessary if `CSSBaseline` is used